adapter/config/types.go

```diff
@@ -208,6 +210,16 @@ type consul struct {
 	KeyFile string
 }
 
+type xdsPayloadFormatter struct {
+	// KeyManagerConfigs contains format configurations related to key manager configuration
+	KeyManagerConfigs keyManagerConfigs
+}
+
+type keyManagerConfigs struct {
```

Comment:
Do we need a new struct ?
```
type xdsPayloadFormatter struct {
	RetainKeysForKeyManagerConfigs []string
}
```

Reply:
Yeah, looks good. But in case we need to add another config for key managers, for example, filter out some keys we can add just another field in TOML for key managers.

```toml
[adapter.xdsPayloadFormatter.keyManagerConfigs]
# Retain only the following keys in the Key Manager configurations from Control Plane events and send to Enforcer
retainKeys = ["self_validate_jwt", "issuer", "claim_mappings", "consumer_key_claim", "scopes_claim", "jwks_endpoint", "certificate_type", "certificate_value", "environments"]
```